pkg/util/k8sutil/backup.go

```diff
@@ -184,9 +184,8 @@ func NewBackupPodSpec(clusterName, account string, sp spec.ClusterSpec) (*v1.Pod
 				Name:  "backup",
 				Image: BackupImage,
 				Command: []string{
-					"/bin/sh",
-					"-ec",
-					"/usr/local/bin/etcd-backup --etcd-cluster=" + clusterName,
+					"/usr/local/bin/etcd-backup",
+					"--etcd-cluster=" + clusterName,
```

Comment:
Can't find a way to comment on that but there is also a /bin/sh in `copyVolume`. Should we replace that too?
https://github.com/hongchaodeng/etcd-operator/blob/c1566b82b9b2307966ebf783a87670f921acf02b/pkg/util/k8sutil/backup.go#L281